api.planx.uk/send/exportZip.ts

```diff
@@ -0,0 +1,207 @@
+import os from "os";
+import path from "path";
+import { $admin } from "../client";
+import { stringify } from "csv-stringify";
+import fs from "fs";
+import str from "string-to-stream";
+import AdmZip from "adm-zip";
+import { deleteFile, resolveStream } from "./helpers";
+import { getFileFromS3 } from "../s3/getFile";
+import {
+  hasRequiredDataForTemplate,
+  generateHTMLMapStream,
+  generateHTMLOverviewStream,
+  generateDocxTemplateStream,
+} from "@opensystemslab/planx-document-templates";
+import { Passport } from "@opensystemslab/planx-core";
+import { Passport as IPassport } from "../types";
+import type { Stream } from "node:stream";
+import type { PlanXExportData } from "@opensystemslab/planx-document-templates/types/types";
+
+export async function buildSubmissionExportZip({
```

Comment:
bonus points: Could we expose this (with XML) at `/admin/session/:sessionId/zip`?